src/Handler/UploadHandler.php

```diff
@@ -93,8 +99,13 @@ public function remove(object $obj, string $fieldName): void
         if ($preEvent->isCanceled()) {
             return;
         }
-
-        $this->storage->remove($obj, $mapping);
+        try {
+            $this->storage->remove($obj, $mapping);
+        } catch (CannotWriteFileException $cannotWriteFileException) {
+            throw $cannotWriteFileException;
```

Comment:
what's the point of throwing the exception and catching it below? Can't we just catch it directly?

Reply:
This exception is not cought today. Again, just trying to follow existing behaviour - but I'm quite fine with catching it.